src/ripple/protocol/Feature.h

```diff
@@ -366,24 +365,23 @@ foreachFeature(FeatureBitset bs, F&& f)
 
 extern uint256 const featureTickets;
 extern uint256 const featureOwnerPaysFee;
-extern uint256 const featureCompareFlowV1V2;
-extern uint256 const featurePayChan;
+extern uint256 const retiredPayChan;
```

Comment:
I think it makes sense to group the unconditionalized amendments together at the bottom, with a comment explaining that they are amendments that have been activated and are live and the "pre-amendment" code has been removed.